source/templates.tex

```diff
@@ -1483,7 +1483,7 @@
 the
 \grammarterm{template-argument}
 for
-\tcode{Array<T>\,::\,op\-er\-a\-tor\,[]\,()}
+\tcode{Array<T>::op\-er\-a\-tor\,[]\,()}
```

Comment:
Please open an editorial issue for this (I thought we already had one, but maybe not.)